dpos/state/arbitratorsmock.go

```diff
@@ -32,6 +32,18 @@ func NewArbitratorsMock(arbitersByte [][]byte, changeCount, majorityCount int) *
 		MajorityCount:               majorityCount,
 		FinalRoundChange:            0,
 		InactiveMode:                false,
+		CurrentReward: RewardData{
```

Comment:
Here can use a NewRewardData method instead of duplicate initializations.

Reply:
Fixed